README.md

```diff
@@ -53,11 +53,10 @@ Readable? Efficient? Can your Python/Ruby/V8 do better?
 Status
 ------
 
-**Lua Fun** is in an early alpha stage. The library fully [documented]
-[Documentation] and covered with unit tests.
+**Lua Fun** is in an early alpha stage. The library fully
```

Comment:
I know, your commit fixes links. But could you fix description too, as you're already here?
"**Lua Fun** is in an early alpha stage." - it is not true, Luafun used as a part of Tarantool in production for many years. I would replace "alpha stage" with "stable".

Reply:
I don't want to change formal project status in such invisible-for-a-user way. I would fix docs first, then resurrect CI and maybe then define another status together with a new release.

Here I want to just fix links.

Reply:
okey to postpone it.